.semaphore/semaphore.yml

```diff
@@ -0,0 +1,51 @@
+version: v1.0
+name: Mast docker build
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+blocks:
+  - name: build
+    task:
+      secrets:
+        - name: codacy_api_token
+        - name: docker
```

Comment:
Just comparing with the skipper changes, I think we need the fdd-github secret too, so that we have $GITHUB_KEY in the deploy script?
https://github.com/fiaas/skipper/commit/1319957a784accce941d9f980b65e38eab77a799#diff-f4c421dab68fb568a2a5f47b7851a85a409574b05f841f635bad1880b82db03bR36

Reply:
Yeah this is the build part, I added it in the deploy

Reply:
Ah, sorry 👍